.github/workflows/ci-main-daily.yml

```
@@ -15,8 +15,8 @@ jobs:
       matrix:
         version:
           - docs
-          - version-2025-2
-          - version-2024-12
+          - version-2025-6
+          - version-2025-4
```

```suggestion
          - version-2025-4
          - version-2025-2
```

docs maps to the "latest" latest, as in currently 2025.6